examples/chip-tool/commands/parse/ManualCodeCommand.h

```diff
@@ -0,0 +1,44 @@
+/*
+ *   Copyright (c) 2020 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#pragma once
+
+#include "../common/Command.h"
+
+#include <setup_payload/SetupPayload.h>
+
+class ManualCodeCommand : public Command
+{
+public:
+    ManualCodeCommand(const char * name) : Command(name)
+    {
+        Argument argInputCode;
+        argInputCode.type  = ArgumentType::String;
+        argInputCode.name  = "decimal-representation";
+        argInputCode.value = reinterpret_cast<void *>(&mDecimalRepresentation);
+        AddArgument("decimal-representation", argInputCode);
```

Comment:
AddArgument("decimal-representation", &mDecimalRepresentation) and use `char * mPIN;`for `mDecimalRepresentation`.
Globally I would like to not use `Argument` directly into the specific commands.

Reply:
Make sense, I can reuse the existing AddArgument  AddArgument("code", &mCode);